Week-2/Exercise2.2/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
+  <title>Google Docs transform</title>
+</head>
+<body>
+  <main class="wrapper">
+    <section class="box_wrapper">
+      <div class="box box1"></div>
+      <div class="box box2"></div>
+      <div class="box box3"></div>
+      <div class="box box4"></div>
+      <h1>Hi!</h1>
+    </section>
+  </main>
+</body>
+</html>
```

Comment:
There is sudden animation on hover out , when it comes back there is no animation on it , please fix that